Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs

```diff
@@ -74,25 +74,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     r =>
                         r.HasOne<ApiClient>().WithMany().HasForeignKey("ApiClientId"));
 
-            modelBuilder.Entity<Application>()
-                .HasMany(a => a.Profiles)
-                .WithMany(a => a.Applications)
-                .UsingEntity("ProfileApplications");
```

Comment:
The `main` branch [still has this](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs). This looks like valuable code. Are you 100% sure we don't need it?